executor/predictor/predictor_process.go

```diff
@@ -139,6 +148,14 @@ func (p *PredictorProcess) feedback(node *v1.PredictiveUnit, msg payload.SeldonP
 
 }
 
+func (p *PredictorProcess) routeFeedback(node *v1.PredictiveUnit, msg payload.SeldonPayload) (int, error) {
```

Comment:
Separate method because in in case of feedback we route it according to information sent by user?
I guess it's worth to add comment about it, as it's not that obvious at first look.

Reply:
Oh I thought it was, this doesn't call the client underneath at all, so it doesn't send any requests.